core/transactions.hxx

```diff
@@ -188,6 +188,9 @@ class transactions : public couchbase::transactions::transactions
 
     void run(const couchbase::transactions::per_transaction_config& config, async_logic&& code, txn_complete_callback&& cb);
 
+    void run(couchbase::transactions::async_txn_logic&& code,
+             couchbase::transactions::async_txn_complete_logic&& complete_cb,
+             const couchbase::transactions::per_transaction_config& cfg = {}) override;
```

Comment:
In ExtSDKIntegration it's no longer per_transaction_config, it's transaction_options.  This and other feedback was raised on https://github.com/couchbaselabs/couchbase-cxx-client/pull/251, which was an expensive full API review, and one I can't afford to redo here  Am I safe in assuming that all feedback on that PR will be used and that the final API will conform to the specced ExtSDKIntegration API from https://hackmd.io/cHIcXWZSQOi23qpJJAa8OQ?